scripts/xcode_local_build

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+set -e
+set -x
```

Comment:
```suggestion
set -euxo pipefail
```

So you're missing `-u` which complains about unused variables (useful for dumb mistakes) and `-o pipefail` which says "if I use a pipe and it errors, then error".  Otherwise, you can have situations like:  

```sh
$ exit 1 | echo 'works'
works
```
when you really want it to error/exit.

Reply:
ty for the feedback! I'll apply these throughout